### PR TITLE
Color temperature for lights

### PR DESCRIPTION
**Pull request by farmio**
added support for absolute and relative color temperature functions of lights.

'tunable_white' sets the relative color temperature in %
'color_temperature' sets the absolute color temperature in Kelvin

this PR supersedes #153

### CODE REVIEW

**Comment by farmio** [comment]
I don't know why the coveralls coverage decreased. Everything I have added should have tests as well. 
I haven't touched the files reported for decreasing coverage: 
xknx/devices/notification.py and 
xknx/devices/action.py

**Comment by Julius2342** [comment]
Coveralls sometims does not like us :)

(im pretty busy with some 35C3 stuff atm, but will have a look at your MR asap...)

**Comment by marvin-w** [comment]
@farmio: Can you please update this PR so that it is up to date with the master branch? :)

File: home-assistant-plugin/custom_components/xknx.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from homeassistant.helpers.event import async_track_state_change
 from homeassistant.helpers.script import Script
 
-REQUIREMENTS = ['xknx==0.8.5']
+REQUIREMENTS = ['xknx==0.9.0']
```
Comment (marvin-w) [inline home-assistant-plugin/custom_components/xknx.py]: please remove this from the PR. The current version will be 0.9.3 and we will merge it from HA directly into XKNX.

File: xknx/devices/remote_value_sensor.py [hunk 2]
```diff
@@ -20,12 +20,13 @@ class RemoteValueSensor(RemoteValue):
     """Abstraction for many different sensor DPT types."""
 
     DPTMAP = {
+        'current': DPTUElCurrentmA,
```
Comment (marvin-w) [inline xknx/devices/remote_value_sensor.py]: thanks for sorting this thats a good idea!

File: tox.ini [hunk 3]
```diff
@@ -8,6 +8,7 @@ setenv =
     PYTHONPATH = {toxinidir}
 commands =
     py.test --cov --cov-report= {posargs}
+whitelist_externals = make
```
Comment (marvin-w) [inline tox.ini]: Why did you add this?

Comment (farmio) [reply tox.ini]: Without it `tox` throws a warning:
`pylint runtests: commands[0] | make pylint
WARNING: test command found but not installed in testenv
  cmd: /usr/bin/make
  env: /Users/meti/Documents/Dev/xknx/.tox/pylint
Maybe you forgot to specify a dependency? See also the whitelist_externals envconfig setting.`
on my system. Same for 
`pydocstyle runtests: commands[0] | make pydocstyle`

**Comment by farmio** [comment]
I have tested it with Home-Assistant 0.84.6 and it works properly with an MDT Led Controller AKD-0424R.02

The only error was 

> File "***/.homeassistant/custom_components/climate/xknx.py", line 72, in <module>
    vol.Optional(CONF_MIN_TEMP): cv.Coerce(float),
AttributeError: module 'homeassistant.helpers.config_validation' has no attribute 'Coerce'

but I haven't touched that file.

**Comment by marvin-w** [comment]
@farmio Thanks for your fast feedback. We somehow need to get rid of those coverage "errors". I already fixed the error in `climate.py` in this PR: https://github.com/home-assistant/home-assistant/pull/19546.

I will merge the changes into our project once the PR is merged.

There seem to be a lot of changes in this PR that are not related to the actual code changes. Can you fix this please?

**Comment by farmio** [comment]
I don't really know how these changes got in here. They are not visible at https://github.com/XKNX/xknx/compare/master...farmio:color-temperature . Maybe something went wrong when I did the `git rebase upstream/master`.
What can I do about it? Shall I create a new PR?

Also I am not entirely sure if ATTR_WHITE_VALUE and SUPPORT_WHITE_VALUE are the best choices for relative color temperature settings. I couldn't find documentation about this back in October. 
It works, but maybe it will conflict with RGBW lights once they get implemented in xknx. I don't really know if tunable white is a thing with RGBW.

File: tox.ini (hunk 3 above)
Comment (farmio) [reply tox.ini]: Could this be the reason the coverage decreased? These warnings occurred with the tests of the last PR #161 in travis too. 
See for example https://travis-ci.org/XKNX/xknx/jobs/471596429 line 589 ff.

**Comment by marvin-w** [comment]
@farmio thanks for updating the PR. I will take another look tomorrow when I'm back home!

**Comment by Julius2342** [comment]
🐬
